src/state/machines/turn.ts

```diff
@@ -1039,6 +1042,14 @@ export const TurnMachine = setup({
               },
             },
             pickingDestination: {
+              always: {
```

Comment:
I believe this implementation is wrong. In the single player rule the player has the `choice` to select their own deck as a destination, but they should still be able to select one of the market decks instead if they would like the plant/animal card to move there

Reply:
![image](https://github.com/user-attachments/assets/e5ebfee4-c37f-45b8-b853-40467bdd7701)

Reply:
fixed